File: aoj/volume100/scala/A10003.scala
object Main {
  def main(args: Array[String]) {
    val sc = new java.util.Scanner(System.in)
    val a = sc.nextInt
    val b = sc.nextInt
    val relation = toRelation(a, b)
    println(relation)
  }

  def toRelation(a: Int, b: Int): String = {
    if (a == b) "a == b"
    else if (a < b) "a < b"
    else if (a > b) "a > b"
    else ""
  }
}

File: aoj/volume100/java/A10003.java
import java.util.Scanner;

class A10003 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int a = scan.nextInt();
        int b = scan.nextInt();

        System.out.println(toRelationship(a, b));
    }

    public static String toRelationship(int a, int b) {
        if (a == b) return "a == b";
        if (a > b) return "a > b";
        if (a < b) return "a < b";
        return "";
    }
}
